openapi.yaml

```diff
@@ -1844,6 +1844,68 @@ paths:
         - oscal_auth:
             - write:ssps
             - read:ssps
+  /system-security-plans/{sspId}/control-implementation/implemented-requirements:
+    post:
+      tags:
+        - OSCAL System Security Plan
+      summary: Add an implemented requirement to an OSCAL system security plan
+      operationId: addImplementedRequirementToSsp
+      parameters:
+        - name: api_key
```

Comment:
Are we including `api_key` on all operations as part of the spec? Maybe @rgauss you can give me a quick yes/no on this.

Reply:
This is really a holdover example.  This and things like the `oscal_auth` references need to be revamped at some point.

Reply:
@hreineck, can we remove these?